lib/puppet/pops/model/factory.rb

```diff
@@ -790,7 +790,9 @@ def self.NUMBER(name_or_numeric)
       end
     else
       # Bad number should already have been caught by lexer - this should never happen
-      raise ArgumentError, "Internal Error, NUMBER token does not contain a valid number, #{name_or_numeric}"
+      #TRANSLATORS 'NUMBER' refers to a method name and the 'name_or_numeric' was the passed in value
```

Comment:
should `NUMBER` and `name_or_numberic` also not be translated, and should that be noted in this comment?

Reply:
done